SConstruct

```diff
@@ -15,7 +15,7 @@ print("CC ",env['CC'])
 print("C++ ",env['CXX'])
 print("Using mngw",env["use_mingw"])
 
-env.Append(CPPPATH=["src/"],LIBS='SDL2')
+env.Append(CPPPATH=["libs/mingw_dev_lib/include"],LIBS='SDL2')
```

Comment:
I tried it and it works for me

Reply:
I tried it and it works for me

Reply:
but if you remove it it does not work??

Reply:
If you remove this line:  "libs/mingw_dev_lib/include" is not working

Reply:
because for me it works without it, the if statement in line 21 "should" already add `"libs/mingw_dev_lib/include"` to the CPPPATH, and only if you're compiling for windows, because those libs are windows specific, right now is not causing any interference, but i would like to know why that `if` statement is failing for you

Reply:
Ah...I made this change because I don't like copying the SDL2 folder into include and putting it inside src, that's all

Reply:
but you shouldn't need to do that at all, SDL2 is never inside src/ is on `libs/mingw_dev_lib/include` besides `CPPPATH=["libs/mingw_dev_lib/include"]` is done in the `if` statement already so why making it "global"?
Are you perhaps using the Visual Studio compiler? that would explain why the if statement is ineffective (which is cool i just didn't know it would work in this configuration)

Reply:
I experimented by modifying

Reply:
That's ok, so are you using the Visual Studio C++ compiler?

Reply:
> That's ok, so are you using the Visual Studio C++ compiler?

I think I'm using the one I installed using scoop

Reply:
do the output from the prints deleted from lines 14~16 confirm that? in windows it defaults to MVSC if there are problems with min_gw

Reply:
Yes, I deleted them because everything was working for me